src/BloomExe/CollectionTab/LibraryView.cs

```diff
@@ -66,6 +66,11 @@ public partial class LibraryView :  UserControl, IBloomTabArea
 			// Set the background image for Mono because the background color does not paint,
 			// and if we override the background paint handler, the default styling of the child 
 			// controls is changed.
+
+			// We are getting an exception if none of the buttons are visible. The tabstrip is set
+			// to Dock.Top which results in the height being zero if no buttons are visible.
+			if ((_toolStrip.Height == 0) || (_toolStrip.Width == 0)) return;
```

Comment:
I wonder whether it would be worth experimenting with setting the control's BackgroundImageLayout, either to Stretch or Tile? If I read the doc correctly (and Mono implements it), either would guarantee that the background works even if the control is resized, and also allow us to use a smaller bitmap, saving memory.
Looks like what you have is workable, though, so I'll go ahead and merge.